<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255,255,255)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>729</width>
         <height>524</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,4">
        <item>
         <widget class="QFrame" name="MainBar">
          <property name="styleSheet">
           <string notr="true">background: #00BCFF;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1,0,1,1">
           <item>
            <widget class="QLineEdit" name="FindBooksEdit">
             <property name="minimumSize">
              <size>
               <width>242</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#FindBooksEdit {
	height: 40px;
	border: none;
	border-radius: 5px;
	padding-left: 10px;
	background: white;
}</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="FindBooksButton">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tìm sách&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="whatsThis">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tìm sách&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="styleSheet">
              <string notr="true">#FindBooksButton{
	height: 40px;
	border-width: 0px;
	border-radius: 5px;
	background-color: #eee;
	color: #222;
	outline: none;
	font: 15pt &quot;Myriad Pro Cond&quot;;
}
#FindBooksButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(157, 157, 157, 255), stop:0.99435 rgba(241, 241, 241, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
             </property>
             <property name="text">
              <string>Tìm sách</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="SignInButton">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">#SignInButton 
{
	border-width: 0px;
	border-radius: 5px;
	font: 15pt &quot;Myriad Pro Cond&quot;;
	background-color: #eee;
	color: #222;
	outline: none;
}
#SignInButton:hover 
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #e9e9e9, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #e9e9e9);
}
#SignInButton:pressed 
{
	padding-top: 2px;
}</string>
             </property>
             <property name="text">
              <string>Đăng nhập</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="SignUpButton">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">#SignUpButton 
{
	border-width: 0px;
	border-radius: 5px;
	font: 15pt &quot;Myriad Pro Cond&quot;;
	background-color: #eee;
	color: #222;
	outline: none;
}
#SignUpButton:hover 
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #e9e9e9, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #e9e9e9);
}
#SignUpButton:pressed 
{
	padding-top: 2px;
}</string>
             </property>
             <property name="text">
              <string>Đăng ký</string>
             </property>
            </widget>
           </item>
          </layout>
          <zorder>frame</zorder>
          <zorder>BooksTable</zorder>
          <zorder>SignUpButton</zorder>
          <zorder>SignInButton</zorder>
          <zorder>FindBooksButton</zorder>
          <zorder>FindBooksEdit</zorder>
          <zorder>horizontalSpacer</zorder>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
          <item>
           <widget class="QTableWidget" name="BooksTable">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>STT</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Tựa sách</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Tác giả</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Nhà xuất bản</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Tình trạng</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="intro"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>FindBooksEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>FindBooksButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FindBooksEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>FindBooksButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
